onnx/shape_inference/implementation.cc

```diff
@@ -227,36 +239,53 @@ static void InferShapesImpl(
           }
       }
   }
+  
   std::vector<std::string> inference_errors;
+  bool has_unsupported_op = false; // check whether exist unsupported ops
+  
   for (auto& n : *g->mutable_node()) {
     // Resolve domain for node
     auto dit = opset_imports.find(n.domain());
     if (dit == opset_imports.end()) {
       continue;
     }
     auto domain_version = dit->second;
-
     const auto schema =
         schema_registry->GetSchema(n.op_type(), domain_version, n.domain());
     InferenceContextImpl ctx(
         n, valueTypesByName, inputDataByName, &graphInferenceContext);    
     if (!schema) {
+      std::cerr << "Warning: Shape inference does not support this operator: "
```

Comment:
May be change the message to "Warning: Unsupported operator " << n.op_type() << ". No schema registered for this operator."

Reply:
Both modified. Thanks for the suggestion.